unistd/dir.c

```diff
@@ -440,6 +441,60 @@ DIR *opendir(const char *dirname)
 }
 
 
+DIR *fdopendir(int filedes)
```

Comment:
Why `int filedes` instead of `int fd` like specified by POSIX? Is `filedes` used in existing code?